README.md

```diff
@@ -1,25 +1,28 @@
 congress-legislators
 ====================
 
-Members of the United States Congress (1789-Present) and congressional committees (1973-Present) in YAML.
+Members of the United States Congress (1789-Present), congressional committees (1973-Present), committee membership (current only), and presidents and vice presidents of the United States in YAML, JSON, and CSV format.
 
-### Using the data
-
-This repository contains data about legislators...:
+[![Build Status](https://secure.travis-ci.org/unitedstates/congress-legislators.png)](http://travis-ci.org/unitedstates/congress-legislators)
```

Comment:
I guess we should probably just start using Circle for all build testing too (in another PR, just noting).